.golangci.yaml

```diff
@@ -1,8 +1,34 @@
 run:
   concurrency: 4
   deadline: 10m
-  go: "1.19"
+
+  skip-files:
+    - "zz_generated\\..*\\.go$"
 
 linters:
   disable:
-    - unused
\ No newline at end of file
+    - unused
+  enable:
+    - revive
+
+issues:
+  exclude-use-default: false
+  exclude:
+    # errcheck: Almost all programs ignore errors on these functions and in most cases it's ok
+    - Error return value of .((os\.)?std(out|err)\..*|.*Close|.*Flush|os\.Remove(All)?|.*printf?|os\.(Un)?Setenv). is not checked
+    # revive:
+    - var-naming # ((var|const|struct field|func) .* should be .*
+    - dot-imports # should not use dot imports
+    - package-comments # package comment should be of the form
+    - unexported-return # exported func .* returns unexported type .*, which can be annoying to use
+    - indent-error-flow # if block ends with a return statement, so drop this else and outdent its block
+    - "exported: (type|func) name will be used as .* by other packages, and that stutters;"
+    # typecheck:
+    - "undeclared name: `.*`"
+    - "\".*\" imported but not used"
+    # allow non-capitalized messages if they start with technical terms
+    - "structured logging message should be capitalized: \"garden(er-apiserver|er-controller-manager|er-admission-controller|er-operator|er-resource-manager|let)"
+  exclude-rules:
+    - linters:
+        - staticcheck
+      text: "SA1019:" # Excludes messages where deprecated variables are used
```

Comment:
```suggestion
      text: "SA1019:" # Excludes messages where deprecated variables are used

```

Reply:
What is the suggested change?

Reply:
No newline at end of file.
Suggested to add new line at end of file